### PR TITLE
Fix #228

### LINKED ISSUE #228: Fix typo in ci README
There is a typo in README for cleanup, should be "destroy" instead of "apply"

### PR DESCRIPTION
**Pull request by bo-quan**
#### What does this PR do?
Fix a typo in AWS ci README

#### Do you have any concerns with this PR?
No

#### How can the reviewer verify this PR?
Run AWS CI cleanup

#### Any background context you want to provide?
No

#### Questions:
- Have you connected this PR to the issue it resolves?
#228 

- Does the documentation need an update?
This PR updates the documentation

- Does this add new Python dependencies?
No

- Have you added unit or functional tests for this PR?
No

- Does this patch update any configuration files?
No

### CODE REVIEW

File: ci/aws/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ terraform apply -auto-approve \
 Should the script either fail or configued not to cleanup, destruction of the
```
Comment (spisarski) [inline ci/aws/README.md]: Oops, another ansible artifact. I guess this should say something like:
Please always destroy your environment after completion...

**Comment by Sanjhana-Jayagopal** [comment]
Run Tests

**Comment by Sanjhana-Jayagopal** [comment]
Run Tests

**Comment by Sanjhana-Jayagopal** [comment]
Run Tests

**Comment by Sanjhana-Jayagopal** [comment]
Run Tests

**Pull request merged**